hr_infraction/models/hr_infraction.py

```diff
@@ -0,0 +1,155 @@
+# -*- coding: utf-8 -*-
+# © 2016 OpenSynergy Indonesia
+# @author Andhitia Rama
```

Comment:
Please add Michael Telahun Makonnen <mmakonnen@gmail.com> in the copyright, this file was just moved in the models...